schema/anyof.go

```diff
@@ -28,3 +30,23 @@ func (v AnyOf) Validate(value interface{}) (interface{}, error) {
 	// TODO: combine errors.
 	return nil, errors.New("invalid")
 }
+
+// Serialize attempts to serialize the value using the first available
+// FieldSerializer which does not return an error. If no appropriate serializer
+// is found, the input value is returned.
+func (v AnyOf) Serialize(value interface{}) (interface{}, error) {
+	for _, serializer := range v {
+		s, ok := serializer.(FieldSerializer)
+		if !ok {
+			continue
+		}
+
+		v, err := s.Serialize(value)
```

Comment:
Should we also try the validator first to make sure it's the one that passed?

Reply:
No, I don't think we should. The Validator is supposed to accept the seralized value, e.g. the IP `"1.2.3.4"` as input but it should not be expected to accept the internal value _returened_ from the Validator (e.g. `[]byte{1,2,3,4}` ) as input. That's the Seralize function's job to check (and error on invalid input values), would you not agree?

Reply:
Make sense.